openfisca_us/tests/policy/baseline/gov/states/in/tax/income/deductions/in_renters.yaml

```
@@ -0,0 +1,22 @@
+- name: Homeowner's renters' deduction, joint, not maxed out
+  period: 2021
+  input:
+    state_code: IN
+    people:
```

Variables get broadcasted across entities, so this works as long as you don't need to specify the hierarchical structure (e.g. different rent amounts for different people).

This should fix the breaking test, since it looks for entities in all the input first-levels if any entity is detected there.
```suggestion
    rent: 2_000
```

---

Thanks @MaxGhenis, that fixed the issue with the renters deduction. I'm having a similar but distinct problem now with the nonpublic school deduction tests. Any recommendations there?

---

OK, I understand why this is failing: it's because by default, people in tests are put into their own group entities. I think we need to add the following to the unit test:
```yaml
people: ...
tax_units:
  tax_unit:
    members: [child1, child2]
```
Probably worth adding an adult head just to be realistic, though seems like the logic should work without it.

---

Thanks, Nikhil. That got this working.